india_compliance/gst_india/constants/custom_fields.py

```diff
@@ -403,7 +403,7 @@ def get_place_of_supply_options(with_other_countries=False):
             "fieldtype": "Data",
             "insert_after": "gst_category",
             "depends_on": "eval:doc.is_transporter",
-            "read_only_depends_on": "eval:doc.gstin",
+            "read_only_depends_on": "eval:doc.gstin && doc.gstin == doc.gst_transporter_id",
```

Comment:
this isn't okay. it should always be editable I guess. what if it is the same but I want to change it?